openmc/tallies.py

```diff
@@ -58,10 +65,6 @@ class Tally(object):
         Type of estimator for the tally
     triggers : list of openmc.trigger.Trigger
         List of tally triggers
-    num_score_bins : Integral
-        Total number of scores, accounting for the fact that a single
-        user-specified score, e.g. scatter-P3 or flux-Y2,2, might have multiple
-        bins
```

Comment:
Seems like this description actually should be listed on `num_scores` rather than what's currently there. Do you agree?